docs/getting-started.md

```diff
@@ -126,14 +126,14 @@ $ onefuzz pools create my-pool linux --query pool_id
 "9e779388-a9c2-4934-9fa2-6ed6f6a7792a"
 ```
 
-To create a managed scaleset of Ubuntu 20.04 VMs using a [general purpose
+To create a managed scaleset of Ubuntu 22.04 VMs using a [general purpose
 Azure VM](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes) that
 belongs to the pool:
 
 ```console
 $ onefuzz scalesets create my-pool $VM_COUNT
 {
-    "image": "Canonical:0001-com-ubuntu-server-focal:20_04-lts:latest",
+    "image": "Canonical:0001-com-ubuntu-server-jammy:22_04-lts:latest",
```

Comment:
What does the jammy indicate?

Reply:
Jammy is the shortened code-name of Canonicals' 22.04 Ubuntu release (full name Jammy Jellyfish), replacing 20.04/Focal Fossa. They use part of the release code name in the image URNs.